app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -3,6 +3,8 @@ module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
       def initialize
+        @validate_and_decode_token_class = Authentication::AuthnJwt::ValidateAndDecodeToken
+
```

Comment:
is the empty line intended?

Reply:
Yes because i preferred to allocate the validate and decode dependencies in a separate function to keep the ctor cleaner